packages/flutter_tools/lib/src/build_system/targets/dart.dart

```diff
@@ -193,6 +193,18 @@ class KernelSnapshot extends Target {
         targetPlatform == TargetPlatform.fuchsia_arm64) {
       targetModel = TargetModel.flutterRunner;
     }
+    // Force linking of the platform for desktop embedder targets since these
+    // do not correctly load the core snapshots in debug mode.
+    // See https://github.com/flutter/flutter/issues/44724
+    bool forceLinkPlatform;
```

Comment:
`bool forceLinkPlatform = false` or set to false in the default case of the switch.

Reply:
Done